runtime/Cpp/runtime/src/Vocabulary.h

```diff
@@ -37,7 +37,7 @@ namespace dfa {
     /// </param>
     /// <seealso cref= #getLiteralName(int) </seealso>
     /// <seealso cref= #getSymbolicName(int) </seealso>
-    Vocabulary(const std::vector<std::string> &literalNames, const std::vector<std::string> &symbolicNames);
+    Vocabulary(std::vector<std::string> literalNames, std::vector<std::string> symbolicNames);
```

Comment:
What's the reason for removing the const references here?

Reply:
Removing the const reference allows the usage of std::move to avoid copying the vector.

```
std::vector<std::string> foo;
Vocabulary(foo); // copy

std::vector<std::string> bar;
Vocabulary(std::move(bar)); // move
```